db/migrations/versions/3a22407701c8_.py

```diff
@@ -0,0 +1,42 @@
+"""empty message
+
+Revision ID: 3a22407701c8
+Revises: 99ad87a4f888
+Create Date: 2023-01-09 16:06:23.980091
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '3a22407701c8'
+down_revision = '99ad87a4f888'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+
+    op.drop_column("flags","resolution_reason")
```

Comment:
Think this is OK for now as no-one is using flags yet, but I think this would delete all data if we had some, so would need to come up with a more incremental migration in future.